node/bft/src/worker.rs

```diff
@@ -58,6 +58,8 @@ pub struct Worker<N: Network> {
     proposed_batch: Arc<ProposedBatch<N>>,
     /// The ready queue.
     ready: Arc<RwLock<Ready<N>>>,
+    /// The priority queue.
+    ready_priority: Arc<RwLock<ReadyPriority<N>>>,
```

Comment:
Architecture question: why duplicate `Ready` instead of duplicating `transactions_queue`?

I see some downsides to duplicating `Ready`:
- Lots of extra lines of code
- Transactions in the `transactions_queue` (of size `1 << 10`) are processed without priority in mind, meaning when there are a lot of transactions, the priority fee will barely accelerate landing your transaction past the ready queues (each of size `100`).

A benefit of using `ready_priority` is that we can pop and push from it, prioritizing transactions with priority fees, but if we assume `ready` was populated with a good ordering in mind, and because it is so small anyway, it doesn't really matter.